src/app/globals.css

```diff
@@ -24,6 +30,31 @@ textarea {
   resize: none;
 }
 
+.absolute{
+  position: absolute;
+}
+.justify-between{
+  justify-content: space-between;
+}
+
+.width-100{
+  width: 100%;
+}
+
+.bg-secondary-blue {
+  background-color: var(--secondary-blue);
+}
+
+/* .bg-light-gray{
+  background-color: var(--light-gray);
+} */
+.padding-container{
```

Comment:
This padding container is a class that can be replaced for p-10 (For padding with 10px) in leal-styles.